From eac928d06a99d11dec4ff1b0422a12d7cfde2302 Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Wed, 2 Sep 2026 23:57:53 -0400
Subject: [PATCH] Refresh the guest package lock

Lock current GNOME archive, Mesa 26.2.2, Vulkan, and WirePlumber patch releases.
---
 guest/packages.lock.json | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index edbd273..3739afd 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -109,7 +109,7 @@
     "glycin": "2.1.5-2",
     "glycin-gtk4": "2.1.5-2",
     "gmp": "6.3.0-3",
-    "gnome-autoar": "0.5.1-1",
+    "gnome-autoar": "0.5.2-1",
     "gnome-desktop-4": "1:44.5-1",
     "gnome-desktop-common": "1:44.5-1",
     "gnome-keyring": "1:50.0-1",
@@ -378,7 +378,7 @@
     "libvterm": "0.3.3-2",
     "libwacom": "2.19.1-1",
     "libwebp": "1.6.0-2",
-    "libwireplumber": "0.5.16-1",
+    "libwireplumber": "0.5.17-1",
     "libx11": "1.8.13-1",
     "libxau": "1.0.12-1",
     "libxcb": "1.17.0-1",
@@ -433,7 +433,7 @@
     "man-db": "2.13.1-2",
     "md4c": "0.5.3-1",
     "mdadm": "4.6-2",
-    "mesa": "1:26.2.1-1",
+    "mesa": "1:26.2.2-1",
     "minizip": "1:1.3.2-3",
     "mkinitcpio": "41.1-1",
     "mkinitcpio-busybox": "1.36.1-1",
@@ -600,14 +600,14 @@
     "vmaf": "3.2.0-1",
     "volume_key": "0.3.12-12",
     "vulkan-icd-loader": "1.4.357.0-1",
-    "vulkan-mesa-implicit-layers": "1:26.2.1-1",
-    "vulkan-swrast": "1:26.2.1-1",
-    "vulkan-virtio": "1:26.2.1-1",
+    "vulkan-mesa-implicit-layers": "1:26.2.2-1",
+    "vulkan-swrast": "1:26.2.2-1",
+    "vulkan-virtio": "1:26.2.2-1",
     "wayland": "1.26.0-1",
     "wayland-protocols": "1.49-1",
     "webrtc-audio-processing-1": "1.3-5",
     "which": "2.25-1",
-    "wireplumber": "0.5.16-1",
+    "wireplumber": "0.5.17-1",
     "wl-clipboard": "1:2.3.0-1",
     "woff2-font-awesome": "7.3.1-1",
     "wpa_supplicant": "2:2.12-1",
-- 
2.55.0

